pages/search/engagement.vue

```diff
@@ -27,11 +27,11 @@
 <script>
 import { mapState } from 'vuex'
 
-import ConShowEngagaments from '@/components/contact/ConShowEngagements.vue'
+import ConShowEngagements from '@/components/contact/ConShowEngagements.vue'
 
 export default {
   components: {
-    ConShowEngagaments
+    'contact-show-engagement': ConShowEngagements
```

Comment:
Nice 👍  makes sense! Much more consistent with the conventions of html element names